retail/retail-java-applications/dashboards/retail_pipeline.json

```diff
@@ -0,0 +1,271 @@
+{
+  "displayName": "Retail Pipeline",
+  "gridLayout": {
+    "columns": "2",
+    "widgets": [
+      {
+        "title": "Total PubSub Acknowledge messages",
+        "xyChart": {
+          "chartOptions": {
+            "mode": "COLOR"
+          },
+          "dataSets": [
+            {
+              "minAlignmentPeriod": "300s",
+              "plotType": "LINE",
+              "timeSeriesQuery": {
+                "timeSeriesFilter": {
+                  "aggregation": {
+                    "crossSeriesReducer": "REDUCE_SUM",
+                    "perSeriesAligner": "ALIGN_RATE"
+                  },
+                  "filter": "metric.type=\"pubsub.googleapis.com/topic/send_message_operation_count\" resource.type=\"pubsub_topic\" resource.label.\"topic_id\"=monitoring.regex.full_match(\"global(.*)\")",
```

Comment:
This should be a script with variables that can be passed in, for example if global is changed as the key word to be used.